tox.ini

```diff
@@ -1,13 +1,22 @@
 [tox]
-envlist = py36,py37,py38,py39,py310,pep8,packaging,noextra
+envlist = py36,py37,py38,py39,py310,pep8,packaging,noextra,mypy
 
 [testenv]
 deps =
     pytest
 commands =
-    pytest --strict {posargs}
+    pytest --strict-markers {posargs}
 extras = md
 
+[testenv:mypy]
+basepython = python3
+deps =
+    mypy
+    types-bleach
+    types-docutils
+    types-Pygments
+commands = mypy --strict readme_renderer
```

Comment:
Glad to see `--strict`. FWIW, I prefer to use an explicit configuration file (ala [twine/mypy.ini](https://github.com/pypa/twine/blob/main/mypy.ini)) and just run `mypy`, because that makes it easier for other tools (e.g. VS Code) to use the same configuration as CI.

Reply:
Agree! I was using this during dev/test and will move to pyproject.yaml where other configs are.